runtime/picasso/Cargo.toml

```diff
@@ -78,15 +78,14 @@ currency-factory = { package = "pallet-currency-factory", path = "../../frame/cu
 governance-registry = { package = "pallet-governance-registry", path = "../../frame/governance-registry", default-features = false }
 primitives = { path = "../primitives", default-features = false }
 vesting = { package = "pallet-vesting", path = "../../frame/vesting", default-features = false }
-
 # Used for the node template's RPCs
 system-rpc-runtime-api = { package = "frame-system-rpc-runtime-api", git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.16", default-features = false }
 transaction-payment-rpc-runtime-api = { package = "pallet-transaction-payment-rpc-runtime-api", git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.16", default-features = false }
 
 # local RPCs
 assets-runtime-api = { path = '../../frame/assets/runtime-api', default-features = false }
 crowdloan-rewards-runtime-api = { path = '../../frame/crowdloan-rewards/runtime-api', default-features = false }
-
+lending-runtime-api = { path = '../../frame/lending/runtime-api', default-features = false }
```

Comment:
thanks for adding. CI will fail until you add std and runtime-benchmarsk dependencies too. just FYI. anyway CI will not pass this code I guess.